src/models/BrowserStyleSheet.js

```diff
@@ -394,6 +394,20 @@ if (!DISABLE_SPEEDY) {
   }
 }
 
+/* Factory for making more tags */
+export function tagConstructorWithTarget(target?: HTMLElement): Function {
+  return function tagConstructor(isLocal: boolean): Tag {
+    const el = document.createElement('style')
+    el.type = 'text/css'
+    el.setAttribute(SC_ATTR, '')
+    el.setAttribute(LOCAL_ATTR, isLocal ? 'true' : 'false')
+    const targ = target || document.head
+    if (!targ) throw new Error('Missing document <head>')
```

Comment:
Actually, we might want to update the error message here :)

Reply:
Adressed!